packages/manager/modules/pci/src/projects/project/storages/databases/components/plans-list/plans-list.html

```diff
@@ -77,6 +77,11 @@
                 class="mb-0"
                 data-translate="pci_database_plans_list_spec_backup"
             ></p>
+            <p
+                ng-if="plan.supportsPrivateNetwork"
```

Comment:
we have to add `data` prefix to keep HTML validation.
```suggestion
                dat-ng-if="plan.supportsPrivateNetwork"
```